docs/TodoList.md

```diff
@@ -329,17 +265,19 @@ return (
   </section>
 );
 ```
+
 Now each TodoItem can display its own text, and perform actions utilizing its own ID. Explaining all the different actions that a TodoItem can invoke in the TodoMVC example goes beyond the scope of this article, but let's just take a look at the action that deletes one of the to-do items. Here is an abbreviated version of the TodoItem:
 
 ```javascript
 var React = require('react');
+var ReactPropTypes = React.PropTypes;
 var TodoActions = require('../actions/TodoActions');
 var TodoTextInput = require('./TodoTextInput.react');
 
 var TodoItem = React.createClass({
 
   propTypes: {
-    todo: React.PropTypes.object.isRequired
+    todo: ReactPropTypes.object.isRequired
```

Comment:
We tend to prefer keeping the name as PropTypes. So either

```
var PropTypes = React.PropTypes;
```

Or if it's supported:

```
var {PropTypes} = React;
```